### PR TITLE
Rename BALO to BALA in Swan Lake Learn Pages

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Rename BALO to BALA in Swan Lake Learn pages.
> Fixes #

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: swan-lake/learn/calling-java-code-from-ballerina.md [hunk 1]
```diff
@@ -278,8 +278,8 @@ Compiling source
 ... 
 ...
 
-Creating balos
-	target/balo/sameera-yaml_package-java11-0.1.0.balo
+Creating balas
```
Comment (azinneera) [inline swan-lake/learn/calling-java-code-from-ballerina.md:281]:
```suggestion
Creating bala
```

File: swan-lake/learn/deployment/code-to-cloud/code-to-cloud-samples.md [hunk 2]
```diff
@@ -66,8 +66,8 @@ $> bal build --cloud=k8s
 Compiling source
         wso2/hello:0.1.0
 
-Creating balos
-        target/balo/hello-2020r2-any-0.1.0.balo
+Creating balas
```
Comment (azinneera) [inline swan-lake/learn/deployment/code-to-cloud/code-to-cloud-samples.md:69]:
```suggestion
Creating bala
```

File: swan-lake/learn/deployment/code-to-cloud/code-to-cloud-samples.md [hunk 3]
```diff
@@ -334,8 +334,8 @@ Execute the `bal build --cloud=k8s` command to build the Ballerina package and y
 Compiling source
         wso2/scaling:0.1.0
 
-Creating balos
-        target/balo/scaling-2020r2-any-0.1.0.balo
+Creating balas
```
Comment (azinneera) [inline swan-lake/learn/deployment/code-to-cloud/code-to-cloud-samples.md:337]:
```suggestion
Creating bala
```

**Comment by praneesha** [comment]
Closing since this is invalid.

**Pull request closed without merging**